File: selection_sort.py
class Solution: 
    def select(self, arr, current_index): 
        min_index = current_index 
        for indx in range(current_index + 1, len(arr)):
            if arr[indx] < arr[min_index]:
                min_index = indx
        return min_index
            
    def selectionSort(self, arr,n):
        for current_index in range(len(arr)):
            min_index = Solution.select(self, arr, current_index)
            arr[current_index],arr[min_index] = arr[min_index], arr[current_index] 

File: selection_sort.java
class Solution
{
	int  select(int arr[], int i)
	{
        // code here such that selectionSort() sorts arr[]
        return i;
	}
	
	void selectionSort(int arr[], int n)
	{
	    //code here
	    int min,temp;
	    for (int i=0; i<n-1; i++){
	        min=select(arr, i);
	        arr[min]=arr[i];
	        for (int j=i+1; j<n; j++){
	            if (arr[j]<arr[min]){
	                min=j;
	            }
	        }
	        temp=arr[i];
	        arr[i]=arr[min];
	        arr[min]=temp;
	    }
	    
	    
	}
}
